src/components/studyList/StudyList.js

```diff
@@ -256,28 +257,28 @@ class StudyList extends Component {
   render() {
     const tableMeta = {
       patientName: {
-        displayText: 'Patient Name',
+        displayText: this.props.t('PatientName'),
         sort: 0,
       },
       patientId: {
-        displayText: 'MRN',
+        displayText: this.props.t('MRN'),
         sort: 0,
       },
       accessionNumber: {
-        displayText: 'Accession #',
+        displayText: this.props.t('Accession'),
         sort: 0,
       },
       studyDate: {
-        displayText: 'Study Date',
+        displayText: this.props.t('StudyDate'),
         inputType: 'date-range',
         sort: 0,
       },
       modalities: {
-        displayText: 'Modality',
+        displayText: this.props.t('Modality'),
         sort: 0,
       },
       studyDescription: {
-        displayText: 'Description',
+        displayText: this.props.t('Description'),
```

Comment:
Can we make the key for this StudyDescription?

Reply:
Done